W-15828285-update-rtf-pod-kt (#1359)
mulesoft/docs-runtime-fabric: modules/ROOT/pages/_partials/include-manage-proxies.adoc

[NOTE]
Starting Runtime Fabric version 2.6.22, the following HTTP proxy is needed for Anypoint Monitoring sidecar or agents as well. SOCK5 proxy is not required for Mule apps deployed on Runtime Fabrics with version 2.6.22 onwards

> The rtf-agent pod and any Mule app that needs the proxy applied requires to be restarted. Although no downtime occurs if you are running multiple replicas, you should perform HTTP proxy updates during a maintenance window. 

Reviewer: ```suggestion
You must restart the `rtf-agent` pod and any Mule app that requires a proxy. Although no downtime occurs if you are running multiple replicas, you should perform HTTP proxy updates during a maintenance window. 
```

Do people know what the rtf-proxy is? I don't see that or "RTF proxy" or "Runtime Fabric proxy" mentioned anywhere else in the docs. I see one mention of a "proxy configuration from Runtime Fabric".

Reviewer: Also not sure if we want to use "should"
